Sync with ballerina 0.990.3 (#54)
wso2-ballerina/module-jira: jira7/Module.md

    }
};

jira7:Client jiraClient = new(jiraConfig);
```

> The `getAllProjectSummaries` function returns the project summary of all the projects if successful or `error` if unsuccessful.

Reviewer: Again let's change "function" to "remote function" wherever applicable.

> The `getAllProjectSummaries` remote function returns the project summary of all the projects if successful or an `error` if unsuccessful.

Reviewer: All these actions seem to be returning `JiraConnectorError` and not `error`, shall we fix all places?
```suggestion
The `getAllProjectSummaries` remote function returns the project summary of all the projects if successful or a `JiraConnectorError` if unsuccessful.
```

> The `getAllProjectSummaries` remote function returns the project summary of all the projects if successful or an `JiraConnectorError` if unsuccessful.

Reviewer: ```suggestion
The `getAllProjectSummaries` remote function returns the project summary of all the projects if successful or a `JiraConnectorError` if unsuccessful.
```

```ballerina
var response = jiraClient->getAllProjectSummaries();

> if (response is ProjectSummary[]) {

Reviewer: Shouldn't this be 
```suggestion
if (response is jira7:ProjectSummary[]) {
```

    io:println("Project Summary: ", response);
} else {
    io:println("Error: ", response);
}
```

> The `createProject` function creates a JIRA project with the given name. It returns `Project` object if successful or `error` if unsuccessful.

Reviewer: ```suggestion
The `createProject` remote function creates a JIRA project with the given name. It returns a `Project` object if successful or `error` if unsuccessful.
```

> The `createProject` remote function creates a JIRA project with the given name. It returns `Project` object if successful or `error` if unsuccessful.

Reviewer: ```suggestion
The `createProject` remote function creates a JIRA project with the given name. It returns a `Project` object if successful or `error` if unsuccessful.
```

> The `createProject` remote function creates a JIRA project with the given name. It returns a `Project` object if successful or an `JiraConnectorError` if unsuccessful.

Reviewer: ```suggestion
The `createProject` remote function creates a JIRA project with the given name. It returns a `Project` object if successful or a `JiraConnectorError` if unsuccessful.
```

```ballerina
var output = jiraClient->createProject("TST_PROJECT");
if (output is jira7:Project) {
    io:println("Project Details: ", output);
} else {
    io:println("Error: ", output.message);
}
```

> The `createIssue` function creates an issue with the given issue details. `IssueRequest` is a object that contains all

Reviewer: ```suggestion
The `createIssue` function creates an issue with the given issue details. `IssueRequest` is an object that contains all
```

> the data that is required to create the issue. It returns `Issue` object if successful or `error` if unsuccessful.

Reviewer: ```suggestion
the data that is required to create the issue. It returns an `Issue` object if successful or `error` if unsuccessful.
```

> the data that is required to create the issue. It returns an `Issue` object if successful or an `JiraConnectorError` if unsuccessful.

Reviewer: ```suggestion
the data that is required to create the issue. It returns an `Issue` object if successful or a `JiraConnectorError` if unsuccessful.
```

    projectId: ”1234”,
    assigneeName: “username”
};

> var output = jiraClient->createIssue(newIssue);

Reviewer: shall we change output to something else?
